server/controllers/community/search.js

```diff
@@ -0,0 +1,62 @@
+const { Op } = require('sequelize');
+const { Profile, User } = require('../../models');
+
+const search = async (req, res) => {
```

Comment:
because we've 2 filter (search)
function name should be more clear :
1- search by name => should be (nameFilter)
2- filter by languages => should be (langFilter)